src/bridge.py

```diff
@@ -38,7 +43,21 @@ def get_status() -> dict:
     return {"is_running": is_running(), "version": version_running}
 
 
+def is_port_in_use(port):
+    """Cheks if a certain port is listening on localhost"""
+    with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
+        return s.connect_ex(("0.0.0.0", port)) == 0
+
+
+def check_bridge_and_proxy_status() -> None:
+    """Reporting the status of bridge and proxy, for debugging purposes"""
+    time.sleep(0.5)
```

Comment:
I think the sleep should be outside the function in `start` as it is not inherently needed from the point of the function. I mean we need it because we are starting/stopping but the function should be a standalone thing which doesn't need it. Does it make sense?